src/main/java/uk/gov/hmcts/reform/adoption/controllers/StartCaseController.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.reform.adoption.controllers;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.adoption.services.CoreCaseDataService;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+
+@RestController
+@RequestMapping("/")
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class StartCaseController {
+    private final CoreCaseDataService coreCaseDataService;
+
+    @PostMapping("start-a-case")
+    public CaseDetails startCase(@RequestHeader(value =  "authorization") String authorisation) {
```

Comment:
In place of individual `@RequestHeader`'s, could we reuse the [RequestData](https://github.com/hmcts/fpl-ccd-configuration/blob/master/service/src/main/java/uk/gov/hmcts/reform/fpl/request/RequestData.java) pattern/impl ?